lib/vmdb/loggers.rb

```diff
@@ -40,7 +40,16 @@ def self.apply_config(config)
     def self.create_loggers
       path_dir = Rails.root.join("log")
 
-      $log               = VMDBLogger.new(path_dir.join("evm.log"))
+      $evm_log = VMDBLogger.new(path_dir.join("evm.log"))
+
+      $log =
+        if ENV["CONTAINER"]
+          stdout_log = StdoutLogger.new.tap { |l| l.formatter = ContainerFormatter.new }
+          MulticastLogger.new($evm_log, stdout_log)
```

Comment:
When looking at the container in Openshift, there is a tab for logs (container STDOUT) where this JSON output woud be visible.  There is a separate tab for the Terminal.